daml-lf/spec/transaction.rst

```diff
@@ -46,14 +49,30 @@ of this document, is not a valid DAML-LF transaction.
 Backward compatibility
 ^^^^^^^^^^^^^^^^^^^^^^
 
-DAML-LF transactions are accompanied by a version identifier; every
-change to ``transaction.proto`` entails a change to this specification,
-and every change to this specification introduces a unique new version.
-`Version history`_ defines a total ordering of all past versions; any
-version *y* unlisted there should be considered *y>k* for any known
-version *k*.
+DAML-LF transaction nodes, and transactions are encoded according a common versioning scheme, called the *transaction version scheme*.
+Each version of this scheme, called a transaction version, is associated to a unique language version.
```

Comment:
Is that true? Isn't `10` assigned to `1.6` up to `1.8`?

Reply:
I drop unique